src/providers/mod.rs

```diff
@@ -52,6 +52,10 @@ use crate::network;
 const ENV_PREFIX: &str = "AFTERBURN_";
 #[cfg(feature = "cl-legacy")]
 const ENV_PREFIX: &str = "COREOS_";
+// AFTERBURN_SSH_AUTHORIZED_KEYS_MESSAGEID keeps track of
+// the journald log related to ssh_authorized_keys information.
+#[cfg(not(feature = "cl-legacy"))]
```

Comment:
I think we can add this logic in all cases, i.e. with and without the legacy feature enabled.

Reply:
Hm, I would have said let's only touch the FCOS path so we don't need to think about handling both, but eh.